docs/meta_roles/pulp_services_without_redis.md

```diff
@@ -0,0 +1,17 @@
+pulp_services_without_redis
```

Comment:
I wonder if we should name it differently, like "pulp_recommended_services" or "pulp_required_services".

We should discuss this over a quick meeting TBH.